flume-ng-channels/flume-kafka-channel/src/test/java/org/apache/flume/channel/kafka/TestParseAsFlumeEvent.java

```diff
@@ -64,9 +65,12 @@ private void doParseAsFlumeEventFalse(Boolean checkHeaders) throws Exception {
     KafkaProducer<String, byte[]> producer = new KafkaProducer<>(props);
 
     for (int i = 0; i < 50; i++) {
+      RecordHeaders headers = new RecordHeaders();
+      headers.add(KEY_HEADER, (String.valueOf(i) + "-header").getBytes());
```

Comment:
What's the significance of `-header` here? Can we make this configurable, leaving `-header` as default?